app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,15 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.Previewer">
+        <activity
+            android:name=".BitmapVideo"
+            android:exported="true" />
+        <activity
+            android:name=".BitmapYoutube"
+            android:exported="true" />
+        <activity
+            android:name=".BitmapPDF"
+            android:exported="true" />
```

Comment:
Set `android:exported="false"` for all these activities. We don't want anyone outside this app to be able of launching these activities. Also, change the activity names to `VideoToBitmapActivity`, `YoutubeVideoToBitmapActivity` and `PdfToBitmapActivity`.

Reply:
done